ts/Grid/Pro/CellRendering/ContentTypes/CheckboxContent.ts

```diff
@@ -61,9 +61,13 @@ class CheckboxContent extends CellContentPro implements EditModeContent {
      *
      * */
 
-    constructor(cell: TableCell, renderer: CheckboxRenderer) {
+    constructor(
+        cell: TableCell,
+        renderer: CheckboxRenderer,
+        parentElement?: HTMLElement
+    ) {
         super(cell, renderer);
-        this.input = this.add();
+        this.input = this.add(parentElement);
```

Comment:
From my perspective it would be better to use `parentElement || this.cell.htmlElement` otherwise you need to take look on `add()` method to understand whats going on, when parentElement is undefined.